internal/helpers/tools.go

```diff
@@ -0,0 +1,10 @@
+// +build tools
```

Comment:
This file is recommended to be able to pin versions of the tools in `go.mod`.
This is why in github-actions and `Dockerfile` we can just do `go mod tidy` to resolve / fetch the versions, then `go install ....` to ensure we all have the same versions.